src/components/internals/calendar-body/calendar-body.styles.js

```diff
@@ -120,37 +133,14 @@ const getDateTimeInputStyles = (props, state) => {
       border-right: none;
       transition: color ${vars.transitionStandard},
         border-color ${vars.transitionStandard};
-
-      &:focus,
-      &:active,
-      &:focus + *,
-      &:active + * {
-        border-color: ${vars.borderColorForInputWhenFocused};
-        color: ${vars.fontColorForInput};
-      }
-
-      &:disabled {
-        cursor: not-allowed;
-      }
-
-      &:disabled,
-      &:read-only {
-        background-color: ${vars.backgroundColorForInputWhenDisabled};
-        color: ${vars.fontColorForInputWhenDisabled};
-        border-color: ${vars.borderColorForInputWhenDisabled};
-        opacity: 1; /* fix for mobile safari */
-      }
```

Comment:
These styles are already applied elsewhere, and having them here creates conflicts with the correct order of the input states.